week2/exercises/04-post-endpoint.md

```diff
@@ -45,7 +46,7 @@ But first, for Express to handle JSON requests, we need to add `app.use(express.
 ```js
 // Contents of app.js
 
-const express = require("express")
+import express from 'express'
 const app = express()
 const port = process.env.PORT || 3000
```

Comment:
I think we should leave the default port in case someone forgets to include .env file

Reply:
Good point. It's unchanged, GH is just showing it in a weird way 😅

Reply:
haha, got it. Good job 💪